cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url()
+    }
+
+    getPageTitle() {
+        return cy.get(".page__title").contains('Timesheets')
+    }
+
+    getCalendarVisable() {
+        return cy.get('.calendar')
+    }
+
+    getCalendarToday() {
+        return cy.get('.calendar--today')
+    }
+
+    getTimeEntryForm() {
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitle() {
+        return cy.get('.user-info__title').contains('Ugne Lekstutiene')
+    }
+
+    getMainLength(tabNumber) {
+        return cy.get('.main-nav').find('li').should('have.length', tabNumber)
+    }
+
+    getTimeLoggingColored() {
```

Comment:
Not sure if this is the correct way to do it, it would probably be better to get not the 'Time Logging' tab, but the active tab. Or if you want to only consider the 'Time Logging' tab, then maybe rename this function to add the color assertion, e.g. `assertTimeLoggingColoredBlue()`